client/coral-ui/components/Tab.js

```diff
@@ -4,7 +4,7 @@ import styles from './Tab.css';
 export default ({children, tabId, active, onTabClick, cStyle = 'base', ...props}) => (
   <li
     key={tabId}
-    className={`${active ? styles[`${cStyle}--active`] : ''} tab ${props.className}`}
+    className={`${active ? `${styles[`${cStyle}--active`]} tab-active` : ''} tab ${props.className}`}
```

Comment:
The only thing that I would suggest is that we should make it slightly more specific with `coral-tab-active` in case there are other tabs at some point. If we add that, this is good to merge 👍